server/src/shared/infra/cache/entities/auth-code-entity.ts

```diff
@@ -0,0 +1,23 @@
+import { RedisEntity } from "../redis-entity"
+
+export class AuthCodeEntity extends RedisEntity {
+  clientId!: string
+
+  userId!: string
+
+  authCodeString!: string
+
+  getEntityKey(){
+    return `${this.constructor.name}/${this.authCodeString}`
+  }
+
+  toJSON(): object {
+    const { clientId, userId, authCodeString, id } = this
+    return {
+      clientId,
+      userId,
+      authCodeString,
+      id
+    }
```

Comment:
does this deliberately exclude the EntityKey?

Reply:
Yep, that's not relevant to the stored json.